### DISCUSSION TITLE
Preview TED API. Notice status is still Submitted

### DISCUSSION THREAD

**Discussion by eduardsmirnov** [OP-TED/eForms-SDK#353] [Q&A] [answered]
I sent my notice through Submit API in Preview environment. The notice does not have a preferred publication date so I expect it to be published asap. After some time of waiting I still see its status as Submitted. How often do you publish notices?

**Comment by eduardsmirnov** [comment, marked as answer]
Sorry, notices were published on workdays around 16:00. 
No problem with it.

**Reply by rengit62** [reply]
Hi Eduard
Can you please tell me the url where you sent the notice ?
I tested the validation url (https://cvs.preview.ted.europa.eu/v1/notices/validation) but when I tried with the url in swagger UI ('https://enotices2.preview.ted.europa.eu/esenders/api/v2/notice/notices/submit) I got 500 error code
( I also tried with Postman setting as bearer token the api-key , getting the same 500 error code )
<!doctype html><html lang="en"><head><title>HTTP Status 500 – Internal Server Error</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>HTTP Status 500 – Internal Server Error</h1></body></html>
Thanks and Regards
Renato Baviera

**Reply by eduardsmirnov** [reply]
Hi Renato, the url is correct https://enotices2.preview.ted.europa.eu/esenders/api/v2/notice/notices/submit
If you experience the problem constantly, I suggest to open a new issue and let TED to investigate it.
